src/styles/star.css

```diff
@@ -0,0 +1,30 @@
+.star-container {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  padding: 16px;
+  box-sizing: border-box;
+}
+
+.star-container__stars {
+  display: flex;
+}
+
+.star-container__stars-new {
+  width: 16px;
+  height: 16px;
+  margin: 0 2px;
+
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.star-container__stars-new:nth-child(-n + 4) {
```

Comment:
let's reuse styles from the stars task, so it would work with any amount of stars (1-5)